docs/contents/components/data-display/table/accessibility.ja.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Table
+description: "`Table`は、列のソートや行の選択やクリックイベント機能を備えたテーブルのコンポーネントです。"
+package_name: "@yamada-ui/table"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`Table`は、アクセシビリティに関して[WAI-ARIA - Table Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/table/)に従います。
+
+`Table`に`aria-label`を設定すると、スクリーンリーダーによって読み上げられます。
+
+```tsx live=false
+<Table columns={columns} data={data} aria-label="DRAGON BALL characters" />
+```
+
+## キーボード操作
+
+| キー            | 説明                                                                                                             | 状態 |
+| --------------- | ---------------------------------------------------------------------------------------------------------------- | ---- |
+| `Tab`           | フォーカスがテーブルに移動すると最初の項目をフォーカスします。テーブル内である場合は次の項目をフォーカスします。 | -    |
+| `Shift` + `Tab` | 無効ではない前の項目をフォーカスします。                                                                         | -    |
+| `ArrowLeft`     | 無効ではない前の項目をフォーカスします。                                                                         | -    |
+| `ArrowRight`    | 無効ではない次の項目をフォーカスします。                                                                         | -    |
+| `ArrowUp`       | 無効ではない前の項目をフォーカスします。                                                                         | -    |
+| `ArrowDown`     | 無効ではない次の項目をフォーカスします。                                                                         | -    |
+| `Home`          | 無効ではない最初の項目をフォーカスします。                                                                       | -    |
+| `End`           | 無効ではない最後の項目をフォーカスします。                                                                       | -    |
+
+## ARIAロールと属性
+
+| コンポーネント          | ロールと属性    | 使い方                                                                                                          |
+| ----------------------- | --------------- | --------------------------------------------------------------------------------------------------------------- |
+| `Table`                 | `role="table"`  | テーブルであることを示します。                                                                                  |
+|                         | `aria-disabled` | `disabledRowIds`が設定されている場合は、`true`を設定します。                                                    |
+|                         | `aria-rowcount` | テーブルの行数を示します。                                                                                      |
+|                         | `aria-colcount` | テーブルの列数を示します。                                                                                      |
+| `Thead` <Tag>内部</Tag> | `aria-sort`     | 表の項目が昇順の場合は`ascending`を設定し、降順の場合は`descending`を設定、指定がない場合は`none`を設定します。 |
```

Comment:
Change `Thead` to `Th`.

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3038/files#diff-2992dbf44702325b1d09cc77277848ac505ae7788a3fc76dbfda934dab7a5ff9R38-R40

https://github.com/yamada-ui/yamada-ui/pull/3038/files#diff-7ad307d16404c4e1c0bfc2852ffc14f4d55552661d2b9e66b357cee2c6012757R38-R40

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3038/files#diff-2992dbf44702325b1d09cc77277848ac505ae7788a3fc76dbfda934dab7a5ff9R40-R42